SPEC.md

```diff
@@ -116,34 +119,62 @@ In this case, the app can reference the name "example.com/trusted-certificate-au
 The dependencies are applied in order and each image dependency can overwrite files from the previous dependency.
 An optional *path whitelist* can be provided, in which case all non-specified files from all dependencies will be omitted in the final, assembled rootfs.
 
-Image Format TODO
+## App Container Executor
 
-* Define the garbage collection lifecycle of the container filesystem including:
+The **App Container Executor** defines the process by which applications contained in ACIs are executed.
+There are two "perspectives" in this process.
+The "*executor*" perspective consists of the steps that the App Container Executor (ACE) must take to set up the environment for the pod and applications.
+The "*app*" perspective is how the app processes inside the pod see the environment.
+
+## Pods
+
+The deployable, executable unit in the App Container specification is the **pod**.
+A **pod** is a list of apps that should be launched together inside a shared context.
+The shared context is defined as the conjunction of the following Linux namespaces (or equivalents on other operating systems):
+- PID namespace
+- network namespace
+- mount namespace
+- IPC namespace
+- UTS namespace
+The context may also consist of one or more isolators.
+
+Each app in a pod will start chrooted into its own unique read-write rootfs before execution.
+
+The definition of the **pod** - namely, the list of constituent apps, and any isolators that should apply to the entire pod - is codified in a [Pod Manifest](#pod-manifest-schema).
```

Comment:
Possible alternatives to manifest for the image. If we called it one of these then 

- metadata file: "An ACI contains the rootfs and the image metadata file"
- file: "An ACI contains the rootfs and the image file"
- spec: "An ACI contains the rootfs and the image spec"
- struct: "An ACI contains the rootfs and the image struct"

I dislike "metadata file" the least because it is a file with metadata about the image it is part of. So the naming would be Image Metadata Schema and Pod Schema.